damus/Views/Zaps/ProfileZapLinkView.swift

```diff
@@ -81,12 +81,13 @@ struct ProfileZapLinkView<Content: View>: View {
                 }
             }
         }
-        .disabled(lnurl == nil)
+        .disabled(lnurl == nil && lud16 == nil)
```

Comment:
This was a bug where it would disable the zap button even if a Lightning address was available but LNURL was not. I'm not sure if I actually experienced this bug personally and don't understand why it hasn't caused issues. This fixes the disabled state such that the zap button is disabled only if neither are available.